light-client/tests/support/single_step/sequential/commit/_less_than_one_third_nil_votes.json

```diff
@@ -0,0 +1,287 @@
+{
```

Comment:
Isn't this supposed to be ignored given the line in the gitignore above? `_*/*`

Reply:
No, the `.gitignore` only ignores the report folders that the test harness creates in case of failing tests. They look like this: `_bisection/report`, `_simple/report`, etc.

Reply:
In fact, because the tests for the light client are now [run with Testgen](https://github.com/informalsystems/tendermint-rs/blob/a1d51fa28eb18dcf8b5935dd6920fb29842c90f1/light-client/tests/light_client.rs#L189), it will ignore any files/folders starting with `_`

Reply:
> IMO it could have been better to disable single test cases in some kind of blacklist. That way the JSON files could have served as a source for test-cases (s.t. we can guarantee the new generated tests to come will contain at least the cases that we already had).
> On the other hand, deleting the failing cases is a bit cleaner.
> 
> FYI @Shivani912 and @andrey-kuprianov

As I wrote above, Testgen will ignore any files or folders starting with `_`. So the easy way to disable a test or a set of tests, but to keep them at the same time, is to rename with `_` in front.